### PR TITLE
RUSThello landing in games-for-redox

### PR DESCRIPTION
**Pull request by HenryTheCat**
Hopefully this should be smooth…

### CODE REVIEW

File: src/rusthello/interface.rs~ [hunk 1]
```diff
@@ -0,0 +1,299 @@
+// This module provides interface functionalities and manages all the input/output part of the program
```
Comment (ticki) [inline src/rusthello/interface.rs~:1]: You commited a swap file.

File: src/rusthello/main.rs~ [hunk 2]
```diff
@@ -0,0 +1,118 @@
+//! A simple Reversi game written in Rust with love
```
Comment (ticki) [inline src/rusthello/main.rs~:1]: Swapfile here too.

Comment (HenryTheCat) [inline src/rusthello/main.rs~:1]: Sorry, apparently git and I don't like each other… Should I do anything to fix that?

Comment (ticki) [inline src/rusthello/main.rs~:1]: Just make a PR removing the swapfiles, and add `*~` to .gitignore.

**Pull request merged**